rlclientlib/live_model_impl.cc

```diff
@@ -244,8 +241,21 @@ namespace reinforcement_learning {
     RETURN_IF_FAIL(live_model_impl::request_multi_slot_decision_impl(event_id, context_json, slot_ids, action_ids, action_pdfs, model_version, status));
 
     RETURN_IF_FAIL(populate_multi_slot_response(action_ids, action_pdfs, std::string(event_id), std::string(model_version), slot_ids, resp, _trace_logger.get(), status));
+
+    if (_learning_mode == LOGGINGONLY)
```

Comment:
I am a bit confused here. I s difference between the LOGGINGONLY and APPRENTICE about patching response before or after logging it? That seems to be true for CB, but in this case resp object is not the one that logged, so either we are not doing what we want, or it is cleaner to concatenate these 2 ifs in one place.

Reply:
As far as rl client is concerned this is the only difference. LOGGINGONLY also doesn't train, but that logic is in the trainer. 

Regarding the two if statements, I agree 100% and I will merge them. In CCB it does not make sense to reset the action order because the list of actions available for each slot is not deterministic, so we just have to make this clear to whoever uses logging only.